Public/Src/Cache/DistributedCache.Host/Configuration/LocalCasServiceSettings.cs

```diff
@@ -107,5 +109,11 @@ public LocalCasServiceSettings()
         /// </summary>
         [DataMember]
         public int? GzipBarrierSizeForGrpcCopies { get; set; } = null;
+
+        /// <summary>
+        /// Thread pool size used by GRPC.
+        /// </summary>
+        [DataMember]
+        public int? GrpcPoolSize { get; set; } = 70;

```

Comment:
I believe the pattern here is to keep it null for nullable types and not to duplicate the default values in many places.